pandas/tests/indexing/test_multiindex.py

```diff
@@ -239,6 +239,15 @@ def test_iloc_getitem_multiindex(self):
             xp = mi_labels.ix['j'].ix[:, 'j'].ix[0, 0]
         assert rs == xp
 
+        # GH8856
+        s = pd.Series(np.arange(10),
+                      pd.MultiIndex.from_product(([0, 1], list('abcde'))))
```

Comment:
can u add several cases using slices

also if other testing of slice u can pull to this section